app/controllers/diagnostics_controller.rb

```diff
@@ -1,12 +1,17 @@
 # frozen_string_literal: true
 
-class DiagnosticsController < ApplicationController
-  before_action :doorkeeper_authorize!, unless: -> { Rails.env.development? }
+class DiagnosticsController < ApiController
```

Comment:
Wasn't sure whether we're getting anything from the ApiController that we don't want?

Reply:
I guess if it works it works 👍

Reply:
the API controller brings in authorisation; the only annoyance is it defaults to json content type which has to be overridden in this controller.